api/app/Traits/Generator/GeneratesUserDoc.php

```diff
@@ -432,6 +464,34 @@ public function experience(Section $section, AwardExperience|CommunityExperience
 
         $this->addLabelText($section, $this->localize('experiences.additional_details'), $experience->details);
 
+        if ($type === WorkExperience::class) {
+            /** @var WorkExperience $experience */
+            if ($experience->employment_category === EmploymentCategory::GOVERNMENT_OF_CANADA->name || $experience->employment_category === EmploymentCategory::CANADIAN_ARMED_FORCES->name) {
+                $experience->loadMissing(['workStreams']);
+                if ($experience->workStreams && count($experience->workStreams) > 0) {
+                    $workStreamsByCommunity = [];
+                    foreach ($experience->workStreams as $workStream) {
+                        if (isset($workStreamsByCommunity[$workStream->community_id])) {
+                            $workStreamsByCommunity[$workStream->community_id]['workStreams'][] = $workStream->name[$this->lang];
+                        } else {
+                            $community = Community::find($workStream->community_id);
+                            $workStreamsByCommunity[$workStream->community_id] = [
+                                'community' => $community->name[$this->lang],
+                                'workStreams' => [$workStream->name[$this->lang]],
+                            ];
+                        }
+                    }
+                    $section->addText($this->localize('common.work_streams'));
+                    collect(Arr::sortRecursive($workStreamsByCommunity))->each(function ($community) use ($section) {
+                        $section->addListItem($community['community'], 0);
+                        foreach ($community['workStreams'] as $workStream) {
+                            $section->addListItem($workStream, 1);
+                        }
+                    });
+                }
+            }
+        }
+
         if ($withSkills) {
             $experience->load(['userSkills' => ['skill']]);
```

Comment:
One little nitpick
If instead of application download you elect to **Download full profile**
Skills appear nested in work streams, which looks confusing. Suggestion, before the skills listing add a section label if non-zero skills

![image](https://github.com/user-attachments/assets/80044a3e-3507-4d73-830f-bec4a28c1426)

Reply:
I agree that skills should have some sort of label, but that seems out of scope for this issue. I would suggest that the label could be Skills and then remove the bold/strong formatting from each skill, but these are questions for @NienkeBr and @GCTC-NTGC/designers.